### PR TITLE
commands: propagate new user to tenantadm 

### PR DESCRIPTION
**Pull request by mchalski**
everything was in place for this to happen, except
correct configuration of useradm.WithTenantVerification
based on the relevant env var.

Issues: MEN-1311

Changelog: title

Signed-off-by: mchalczynski <marcin.chalczynski@rndity.com>

@maciejmrowiec @mendersoftware/rndity

### CODE REVIEW

File: commands.go [hunk 1]
```diff
@@ -21,6 +21,7 @@ import (
 	"github.com/mendersoftware/go-lib-micro/config"
 	"github.com/mendersoftware/go-lib-micro/identity"
 	"github.com/mendersoftware/go-lib-micro/log"
+	"github.com/mendersoftware/useradm/client/tenant"
```
Comment (bboozzoo) [inline commands.go]: can you move this import to the block with other `github.com/mendersoftware/useradm/` imports?

Comment (mchalski) [reply commands.go]: fixed

**Comment by maciejmrowiec** [comment]
@ping ;)

**Comment by bboozzoo** [comment]
needs a rebase now

**Comment by mchalski** [comment]
@maciejmrowiec @mendersoftware/rndity @GregorioDiStefano 

this PR needs a new one: https://github.com/mendersoftware/integration/pull/360.

I tested them together here, successfully: https://mender-jenkins.mender.io/job/mender-builder/2482/console

Someone please take a look at both and force a merge.

**Pull request merged**